src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -1,108 +1,127 @@
 package org.codewars.utils;
 
 import java.math.BigInteger;
+import java.util.InputMismatchException;
 
 public class TaskRunner {
-    public static void setUser(int id){
+
+    private final Users user;
+
+    public static void setUser(int id) {
         // TODO: Implement setUser() method
     }
+
     public static long[] runGap() {
         return new long[0];
     }
 
-    public static BigInteger runPerimeter(){
+    public static BigInteger runPerimeter() {
         return new BigInteger("0");
     }
 
-    public static int runZeros(){
-       return 0;
-    }
-
-    public static double runF(){
-        return 0.0;
-    }
-
-    public static long runFindNb(){
+    public static int runZeros() {
         return 0;
     }
 
-    public static double runMean(){
-        return 0;
```

Comment:
Do you really want to delete it?

Reply:
This is a task with two methods. So we combined them into one. What do you recommend, maybe we should create a new branchfor new method ?